tests/functional-all.js

```diff
@@ -16,6 +16,7 @@ define([
   "./functional/comments-non-auth.js",
   "./functional/contributors-non-auth.js",
   "./functional/comments-auth.js",
+  "./functional/image-uploads-auth.js",
```

Comment:
Can we change the name of this file to something like `comments-image-uploads-auth.js`? Initially I was a bit confused why we needed authed upload tests (but also, I'm very slow 🐢...)